src/actions/lotesActions.js

```diff
@@ -0,0 +1,31 @@
+import { Firebase, FirebaseRef } from '../lib/firebase';
+
+/**
+ * Set an Error Message
+ */
+export function setError(message) {
+  return dispatch => new Promise(resolve => resolve(dispatch({
+    type: 'LOTES_ERROR',
+    data: message,
+  })));
+}
+
+/**
+ * Get Lotes
+ */
+export function getLotes(activeLotes) {
+  if (Firebase === null) {
+    return () => new Promise(resolve => resolve());
+  }
+
+  return dispatch => new Promise(resolve => FirebaseRef.child('lotes').orderByChild('title')
+    .on('value', (snapshot) => {
+      const lotes = snapshot.val() || {};
+
+      return resolve(dispatch({
+        type: 'LOTES_REPLACE',
+        data: lotes,
+        activeLotes // pass activeLotes to loteReducer as action.activeLotes
```

Comment:
Why does `activeLotes` go thru this function if it doesn't do anything in the function or affect anything on it?

Reply:
Oh, I see we are using that value to filter lotes in the reducer but we don't need to pass this to the reducer because the reducer has access to the whole store state and we could get `activeLotes` from there.